src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,42 @@
 import React from 'react';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+import { PostPrepared } from '../../types/postsPrepared';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+interface Props {
+  post: PostPrepared,
+}
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  if (!post.user) {
+    throw new Error('The author of the post was not found');
+  }
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          <UserInfo key={post.userId} user={(post.user)} />
+        </p>
```

Comment:
If text `posted by` rendered only for userinfo maybe would be better to move it inside `UserInfo` component ?

Why are you rendering component user when it doesn't exist ?
Let's add check for `user` here before render it.

```suggestion
          {user && <UserInfo key={post.userId} user={user} />}
```